### DISCUSSION TITLE
On local build, using yarn build and yarn start, will the public npm be used?

### DISCUSSION THREAD

**Discussion by goldzulu** [FlowiseAI/Flowise#255] [Q&A]
I am testing a modified version of the flowise-ui package in local and has renamed the ui package to use mytestpackage-ui instead of flowise-ui. 

Using yarn dev it pickup the changes (very simple change to test). However when running a full yarn build and yarn start, it seems it is picking the package from node_modules for some reason rather then the local packages/ui directory.

I am not sure why this is happening and any pointers will be appreciated. The mytestpackage-ui is done by simply renaming the package.json in packages/ui to say mytestpackage-ui version 0.0.1.

**Comment by HenryHengZJ** [comment]
you can replacing this line to your local package - https://github.com/FlowiseAI/Flowise/blob/main/packages/server/package.json#L55
